presto-spi/src/main/java/com/facebook/presto/spi/session/SessionPropertyConfigurationManagerFactory.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.session;
+
+public interface SessionPropertyConfigurationManagerFactory
```

Comment:
The name of this class is too long and makes me spend some time parsing it :) But, I cannot really find a better name, if you have some ideas about making this shorter it would be great.

Reply:
I tried! Let me know if you can think of a better one... It seems to be that removing any of "session", "property", "configuration", "manager" and "factory" would be wrong. Maybe we can change "configuration" to "config"? But that doesn't help much, and would be inconsistent with ResourceGroupConfigurationManagerFactory

Reply:
It's consistent with `ResourceGroupConfigurationManagerFactory` (which I also feel is too long...)